src/Components/Home/NewQuestionModal.js

```diff
@@ -2,8 +2,26 @@ import React from "react";
 import { Modal, Form, Button } from "react-bootstrap";
 
 const NewQuestionModal = (props) => {
+  
+  const confirmAnswer = () => {
+    props.onSaveClick({
+      productId: props.productId,
+      question: props.question,
+      answer: props.answer,        
+    })
+  }
+
+  const handleKeyboardEvent = (e) => {
+    if(!!props.question && e.keyCode === 13){
+      e.preventDefault();
+      if(props.answer.length !== 0){
+        confirmAnswer();
+      }
+    }
+  }
+
```

Comment:
It is a good catch and seems like a reasonable fix. I don't know whether there is another way to approach this.

As for the code cleanness:  
1) we now have `!!props.question && props.answer.length === 0` in two places - would be nice to have a const for this e.g. hasEmptyAnswer 
2) 13 is a magic string, though a well-know one, but nevertheless. but can also warrant a const e.g. EnterKeyCode

There two are not critical, up to you.

Reply:
Good points. I'll implement both of these.

Reply:
actually I don't see the duplication of `!!props.question && props.answer.length === 0` so have left that for now

Reply:
an old one is at line 68, a new one is split between lines 15 and 17 as visible in the code snippet above.  it is more visible if !!props.question is moved into the second if. which doesn't ruin any logic - we can always prevent default for Enter key press, and process confirmation if question is defined and answer is not empty.